Note: I tried to just use something simpler like 
```
if defined(JPEGXL_NUMERIC_VERSION) || JPEGXL_NUMERIC_VERSION < JPEGXL_COMPUTE_NUMERIC_VERSION(0, 9, 0)
```
But it does not work because the preprocessor gives a syntax error when trying to parse ` JPEGXL_COMPUTE_NUMERIC_VERSION(0, 9, 0)` because `JPEGXL_COMPUTE_NUMERIC_VERSION` it is undefined.

```
Source/WebCore/platform/image-decoders/jpegxl/JPEGXLImageDecoder.cpp:433:97: error: missing binary operator before token "("
  433 | #if !defined(JPEGXL_NUMERIC_VERSION) || (JPEGXL_NUMERIC_VERSION < JPEGXL_COMPUTE_NUMERIC_VERSION(0, 9, 0))
      |                                                                                                 ^
```

--- a/Source/cmake/OptionsGTK.cmake
+++ b/Source/cmake/OptionsGTK.cmake
@@ -419,7 +419,7 @@ if (ENABLE_WAYLAND_TARGET)
 endif ()
 
 if (USE_JPEGXL)
-    find_package(JPEGXL)
+    find_package(JPEGXL 0.7.0)
     if (NOT JPEGXL_FOUND)
         message(FATAL_ERROR "libjxl is required for USE_JPEGXL")
     endif ()

--- a/Source/cmake/OptionsWPE.cmake
+++ b/Source/cmake/OptionsWPE.cmake
@@ -238,7 +238,7 @@ if (ENABLE_SPEECH_SYNTHESIS)
 endif ()
 
 if (USE_JPEGXL)
-    find_package(JPEGXL)
+    find_package(JPEGXL 0.7.0)
     if (NOT JPEGXL_FOUND)
         message(FATAL_ERROR "libjxl is required for USE_JPEGXL")
     endif ()

